source/server-discovery-and-monitoring/tests/unified/logging-replicaset.yml

```diff
@@ -0,0 +1,269 @@
+        ignoreExtraLogs: true
+        ignoreLogMessages:
+          - data:
+              message: "Server heartbeat started"
+          - data:
+              message: "Starting server monitoring"
+          - data:
+              message: "Stopped server monitoring"
+          - data:
+              message: "Topology description changed"
+        messages:
+          - level: debug
+            component: topology
+            data:
+              message: "Starting topology monitoring"
+              topologyId: { $$exists: true }
+          - level: debug
+            component: topology
+            data:
+              message: "Server heartbeat succeeded"
+              awaited: { $$exists: true }
+              topologyId: { $$exists: true }
+              serverHost: { $$type: string }
+              serverPort: { $$type: [int, long] }
+              driverConnectionId: { $$exists: true }
+              serverConnectionId: { $$exists: true }
+              durationMS: { $$type: [int, long] }
+              reply:
+                $$matchAsDocument:
+                  "$$matchAsRoot":
+                    ok: 1
+          - level: debug
+            component: topology
+            data:
+              message: "Server heartbeat succeeded"
+              awaited: { $$exists: true }
+              topologyId: { $$exists: true }
+              serverHost: { $$type: string }
+              serverPort: { $$type: [int, long] }
+              driverConnectionId: { $$exists: true }
+              serverConnectionId: { $$exists: true }
+              durationMS: { $$type: [int, long] }
+              reply:
+                $$matchAsDocument:
+                  "$$matchAsRoot":
+                    ok: 1
+          - level: debug
+            component: topology
+            data:
+              message: "Server heartbeat succeeded"
+              awaited: { $$exists: true }
+              topologyId: { $$exists: true }
+              serverHost: { $$type: string }
+              serverPort: { $$type: [int, long] }
+              driverConnectionId: { $$exists: true }
+              serverConnectionId: { $$exists: true }
+              durationMS: { $$type: [int, long] }
+              reply:
+                $$matchAsDocument:
+                  "$$matchAsRoot":
+                    ok: 1
+  - description: Failing hearbeat
+    operations:
+    - name: createEntities
+      object: testRunner
+      arguments:
+        entities:
+        - client:
+            id: *client
+            observeLogMessages:
+              topology: debug
+            observeEvents:
+            - serverHeartbeatStartedEvent
+            - serverHeartbeatFailedEvent
+            uriOptions:
+              heartbeatFrequencyMS: 500
+              appname: failingHeartbeatLoggingTest
+    - name: failPoint
+      object: testRunner
+      arguments:
+        failPoint:
+          configureFailPoint: failCommand
+          mode:
+            times: 2
```

Comment:
why 2x? (leave a comment? same for other similar tests)

Reply:
Accounting for RTT thread. Done.